### PR TITLE
Add GitHub action to run when issues are opened

### LINKED ISSUE #1540: Add automation to add new issues to Delivery Assurance board
**Point of contact on this issue**
@Sgtpluck 

**Billable?**

- [x] Yes
- [ ] No
If yes, tock code:
1813

**Skills needed**

- [ ] Any human
- [ ] Design
- [ ] Content
- [x] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [x] Yes
- [ ] No

How much time do you anticipate this work taking?
1/2 day

### A description of the work
In order to get the Delivery Assurance shared board working automatically, we need to add a GitHub action to the Tock repo.

Example Action here:
https://github.com/18F/ux-guide/blob/main/.github/workflows/on_new_issue.yml

This work includes adding secrets to the tock repo.

### Acceptance Criteria

As an 18F staff member who opens an issue on the Tock repo, I can immediately see that issue in the Delivery Assurance [shared Projects board](https://github.com/orgs/18F/projects/41/views/1?query=is%3Aopen+sort%3Aupdated-desc).

This ticket is part of https://github.com/18F/delivery-assurance/issues/9

### PR DESCRIPTION
**Pull request by Sgtpluck**
## Description

This code adds a GitHub Action that runs when a new issue is opened in this repo.

The GitHub Action populate the issue onto the Delivery Assurance [projects board](https://github.com/orgs/18F/projects/41) without a status where it will be ready to be prioritized or removed.

This is part of https://github.com/18F/delivery-assurance/issues/9 and closes #1540

### CODE REVIEW

**Review by cantsin — APPROVED** [review]
LGTM. (Just checked to make sure the secrets were already in place; they are)

**Comment by Sgtpluck** [comment]
@cantsin just updated with label exclusion -- do you wanna look real quick? thanks!

File: .github/workflows/on-add-issue.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          app_id: ${{ secrets.PROJECT_APP_ID }}
+          installation_id: ${{ secrets.PROJECT_INSTALLATION_ID }}
+          private_key: ${{ secrets.PROJECT_PRIVATE_KEY }}
+      # Now we can add the issue to the project board.
+      - name: add to project board
+        uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/18F/projects/41
+          github-token: ${{ steps.app_token.outputs.token }}
+          labeled: compliance
```
Comment (cantsin) [inline .github/workflows/on-add-issue.yml:29]: Hm. We had this label as `Compliance` and I'm not sure if the comparison is case sensitive. Do you know? Eh,  I just changed it to the lowercase version, so either way this should work.

**Review by cantsin — APPROVED** [review]
LGTM

**Pull request merged**